### PR TITLE
docs: Added ace documentation page

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary
Short ace documentation

### Location
https://docs-git-fork-frankkopp-ace-docs-flybywire.vercel.app/dev-corner/development-projects/ace/

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

File: docs/dev-corner/development-projects/.pages [hunk 1]
```diff
@@ -1,6 +1,8 @@
 title: FlyByWire Projects
 nav:
     - Overview: index.md
-    - Website Project: website.md
     - Documentation Project: documentation.md
+    - Website Project: website.md
+    - Discord Bot: discord-bot.md
+    - ace: ace.md
```
Comment (Valastiri) [inline docs/dev-corner/development-projects/.pages]:
```suggestion
    - Ace: ace.md
```
Should probably capitalize page titles? 

Could also change the title to the product name? "Advanced Cockpit Emulator"

File: docs/dev-corner/development-projects/ace.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```json
+{
+	"name": "a32nx",
+	"paths": {
+		"instrumentSrc": "src\\instruments\\src",
+		"bundlesSrc": "src\\instruments\\aceBundles",
+		"htmlUiSrc": "flybywire-aircraft-a320-neo\\html_ui"
+	}
+}
+```
```
Comment (Valastiri) [inline docs/dev-corner/development-projects/ace.md]: Maybe add titles to the code blocks? While not 100% necessary it does look nicer :) 
```suggestion
```json title="sample title here"
{
	"name": "a32nx",
	"paths": {
		"instrumentSrc": "src\\instruments\\src",
		"bundlesSrc": "src\\instruments\\aceBundles",
		"htmlUiSrc": "flybywire-aircraft-a320-neo\\html_ui"
	}
}
```
```

**Review by Valastiri — COMMENTED** [review]
Suggestions

File: docs/dev-corner/development-projects/.pages (hunk 1 above)
Comment (frankkopp) [reply docs/dev-corner/development-projects/.pages]: That is the question - I think it looks strange in small letters - it should be ACE.
@Benjozork what do you think - you had it in small letters.

**Pull request merged**